src/react-apps/applications/service-development/__tests__/DeployToTest.test.tsx

```diff
@@ -0,0 +1,162 @@
+/* tslint:disable:object-literal-key-quotes */
+/* tslint:disable:no-string-literal */
+import { mount } from 'enzyme';
+import 'jest';
+import * as React from 'react';
+
+jest.mock('../../shared/src/version-control/versionControlHeader');
+import { DeployToTestContainer } from '../src/features/deploy/containers/deployToTestContainer';
+
+
+describe('Deploy To Test container', () => {
+  let mockClasses: any;
+  let mockDeploymentList: any;
+  let mockLanguage: any;
+  let mockMasterRepoStatus: any;
+  let mockRepoStatus: any;
+
+  beforeEach(() => {
+    mockClasses = {};
+    mockLanguage = {};
+    mockRepoStatus = {
+      behindBy: 0,
+      aheadBy: 0,
+      contentStatus: [],
+      repositoryStatus: 'Ok',
+      hasMergeConflict: false,
+    };
+
+    mockDeploymentList = {
+      'at21': {
+        'items': [
+          {
+            'spec': {
+              'template': {
+                'spec': {
+                  'containers': [
+                    {
+                      'name': 'example-repo',
+                      'image': 'tddregistry.azurecr.io/tdd-helloworld:1',
+                    },
+                  ],
+                },
+              },
+            },
+          },
+        ],
+      },
+    };
+
+    mockMasterRepoStatus = {
+      'commit': {
+        'id': '2',
+      },
+    };
+  });
+
+  it('should render "Ready for deploy and all checks passed"', async () => {
+    const wrapper = mount(
+      <DeployToTestContainer
+        classes={mockClasses}
+        deploymentList={mockDeploymentList}
+        language={mockLanguage}
+        masterRepoStatus={mockMasterRepoStatus}
+        repoStatus={mockRepoStatus}
+      />,
+    );
+
+    expect(wrapper.text()).toMatch('Du har delt dine endringer');
+
+    // Assert renderRepoInSync part (local and master is in sync)
+    expect(wrapper.exists('#renderInSync')).toEqual(true);
+    const renderInSync = wrapper.find('#renderInSync');
```

Comment:
Does this acctually work? Won't the wrapper just consist of language keys since mockLanguage = {}?
Also I dont agreed that this should be part of a test, tests should not look for specific text strings

Reply:
#renderinsync is an html element ID.

Reply:
comment is on expect(wrapper.text()).toMatch('Du har delt dine endringer');
somehow it got moved to the wrong place

Reply:
Ah, that will be removed and replaced with language keys.

Reply:
good 👍